reference/constraints/False.rst

```diff
@@ -1,120 +1,9 @@
-
-    protected $state;
-
-    protected $invalidStates = array();
-
-    public function isStateInvalid()
-    {
-        return in_array($this->state, $this->invalidStates);
-    }
-
-In this case, the underlying object is only valid if the ``isStateInvalid``
-method returns **false**:
-
-.. configuration-block::
-
-    .. code-block:: php-annotations
-
-        // src/AppBundle/Entity/Author.php
-        namespace AppBundle\Entity;
-
-        use Symfony\Component\Validator\Constraints as Assert;
-
-        class Author
-        {
-            /**
-             * @Assert\False(
-             *     message = "You've entered an invalid state."
-             * )
-             */
-             public function isStateInvalid()
-             {
-                // ...
-             }
-        }
-
-    .. code-block:: yaml
-
-        # src/AppBundle/Resources/config/validation.yml
-        AppBundle\Entity\Author
-            getters:
-                stateInvalid:
-                    - 'False':
-                        message: You've entered an invalid state.
-
-    .. code-block:: xml
-
-        <!-- src/AppBundle/Resources/config/validation.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
-
-            <class name="AppBundle\Entity\Author">
-                <getter property="stateInvalid">
-                    <constraint name="False">
-                        <option name="message">You've entered an invalid state.</option>
-                    </constraint>
-                </getter>
-            </class>
-        </constraint-mapping>
-
-    .. code-block:: php
-
-        // src/AppBundle/Entity/Author.php
-        namespace AppBundle\Entity;
-
-        use Symfony\Component\Validator\Mapping\ClassMetadata;
-        use Symfony\Component\Validator\Constraints as Assert;
-
-        class Author
-        {
-            public static function loadValidatorMetadata(ClassMetadata $metadata)
-            {
-                $metadata->addGetterConstraint('stateInvalid', new Assert\False());
-            }
-        }
-
 .. caution::
 
-    When using YAML, be sure to surround ``False`` with quotes (``'False'``)
-    or else YAML will convert this into a ``false`` boolean value.
-
-Options
--------
-
-message
-~~~~~~~
-
-**type**: ``string`` **default**: ``This value should be false.``
-
-This message is shown if the underlying data is not false.
+    The ``False`` constraint is deprecated since Symfony 2.7
+    and will be removed in Symfony 3.0. Use the ``IsFalse`` constraint instead.
 
-.. include:: /reference/constraints/_payload-option.rst.inc
+.. include:: /reference/constraints/IsFalse.rst
```

Comment:
instead of including the IsFalse documentation, I prefer to add a link to this docs in the caution above. So we end up with:

```rst
.. caution::

    The ``False`` constraint is deprecated since Symfony 2.7 and will
    be removed in 3.0. Use the :doc:`/reference/constraints/IsFalse` constraint
    instead.
```